### DISCUSSION TITLE
Alert when a specific person replies or mentions you

### DISCUSSION THREAD

**Discussion by Yallow-exe** [rigwild/discord-self-bot-console#59] [Q&A]
I would like to know if it is possible to know who mentions you, whether in a message or in replying to a message. I would like it to make it precise, such as triggering the alert only when it is a special person who mentions by getting id or author username..

**Comment by rigwild** [comment]
You can take inspiration from https://github.com/rigwild/discord-self-bot-console/discussions/33

**Comment by DeadLyBro** [comment]
And you can use https://github.com/rigwild/discord-self-bot-console/discussions/29#discussioncomment-3351612 to make beep sound for special user. Actually i think you are looking for that comment. ;)
